src/executor/source/executor/nodes/ScriptNode.ts

```diff
@@ -10,7 +10,7 @@ export interface ScriptNode {
  *
  * @param node to execute
  * @param inputs of the node as an object, with the handle ids as the keys and the inputs as the values
- * @returns the output of the JavaScript function
+ * @returns the output of the JavaScript function wraped into an object in fields of the outgoingHandle IDs
```

Comment:
typo: `wrapped`. Also: Call it `inputs` everywhere if there might be multiple inputs (check other files too please).